swap_meet/vendor.py

```diff
@@ -0,0 +1,96 @@
+from swap_meet.item import Item
+
+#Wave 1 
+class Vendor:   
+    def __init__(self, inventory = None):
+        if inventory == None:
+            self.inventory = []
+        else: 
+            self.inventory = inventory
+    
+    def add(self,item):
+        self.inventory.append(item)
+        return item
+
+    def remove(self,item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        else:
+            return False
+
+#Wave 2
+    def get_by_category(self,category):
+        category_list = []
+
+        for item in self.inventory:
+            if item.category == category:
+                category_list.append(item)
+        return category_list
+
+#Wave 3:
+    def swap_items(self,friend, my_item, their_item):
+        #if len(self.inventory) == 0 or len(friend.inventory) ==0:
+            #return False
+        if my_item in self.inventory and their_item in friend.inventory:
```

Comment:
Consider refactoring so you check the condition where you return `False` first, and then move the main logic out of the conditional.